packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -427,10 +440,12 @@ export class PipelineEditor extends React.Component<
         }
 
         showDialog({
-          title: 'Run pipeline',
-          body: new PipelineSubmissionDialog({ runtimes: response.runtimes }),
+          title: 'Export pipeline',
+          body: new PipelineExportDialog({
+            runtimes: response.runtimes
+          }),
           buttons: [Dialog.cancelButton(), Dialog.okButton()],
-          focusNodeSelector: '#pipeline_name'
+          focusNodeSelector: '#runtime_config'
```

Comment:
I believe this is still needed, as the default value is not calculated by guess and thus if you don't have metadata named "kfp", it will not work.

Reply:
I understand what you're saying, I just disagree since we're setting a value (`kfp`) in the code, even if that value is "bad" but I don't feel strongly about this and am ok leaving this line in if you think it we should focus on a field in the form rather than the ok button.

Reply:
I didn't touch this in my commit, will resolve since I have no string feelings and you seem to have at least some.